doc/installationguide.md

```diff
@@ -14,6 +14,14 @@ These are the parameters that can be configured in the global section:
 	port: '1026'
```

Comment:
An entry about the changes introduces in this PR should be added to CHANGES_NEXT_RELEASE. Suggestion:

```
- Add: allow NGSIv2 for updating active attributes at CB, through configuration (#527)
```

or a similar wording.

Reply:
Done